src/containers/EnterpriseApp/index.jsx

```diff
@@ -42,26 +63,25 @@ class EnterpriseApp extends React.Component {
 
     return (
       <div>
-        {enterpriseId &&
```

Comment:
We no longer need to verify that the `enterpriseId` exists before rendering the redirect/routes as it always will exist.